app/gui2/src/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -132,35 +144,71 @@ const SHAPE_TO_SYMBOL: Record<string, d3.SymbolType> = {
   triangle: d3.symbolTriangle,
 }
 
-const SCALE_TO_D3_SCALE: Record<ScaleType, () => d3.ScaleContinuousNumeric<number, number>> = {
+const SCALE_TO_D3_SCALE: any = {
```

Comment:
Please don't use `any` type. The previous one should work fine here, right?

Reply:
oh yes sorry I must have just missed this!